inventree_kicad/serializers.py

```
@@ -285,15 +285,30 @@ def get_custom_fields(self, part, excluded_field_names):
-            ki_visible_fields = [field.strip().lower() for field in ki_visible_fields]
+        # Find the value parameter value associated with this part instance
+        template_id = self.plugin.get_setting('KICAD_FIELD_VISIBILITY_PARAMETER', None)
+        kicad_local_field_visibility = None
+        try:
+            # check if local parameter set, if so extract fields that need displaying in KiCad
+            kicad_local_field_visibility = self.get_parameter_value(part, template_id, None).split(',')
+
+            # make lower case and strip
+            kicad_local_field_visibility = [field.strip().lower() for field in kicad_local_field_visibility]
+
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)